src/platform/qt/CheckBoxDelegate.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2013-2022 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#pragma once
+
+#include <QStyledItemDelegate>
+
+namespace QGBA {
+
+class CheckBoxDelegate : public QStyledItemDelegate
+{
+public:
```

Comment:
No `Q_OBJECT`?

Reply:
Nope, don't need it. No signals, no slots, no qobject_cast, no QMetaType. The superclass's implementation takes care of everything in that regard.